ampd/src/handlers/solana_verify_worker_set.rs

```diff
@@ -194,10 +196,17 @@ where
 
         let gw_event = parse_gateway_event(&sol_tx).map_err(|_| Error::DeserializeEvent)?;
 
-        let pub_key = match gw_event {
-            GatewayEvent::OperatorshipTransferred {
-                info_account_address,
-            } => info_account_address,
+        let operators = match gw_event {
+            GatewayEvent::OperatorshipTransferred(Cow::Owned(TransferOperatorshipCommand {
+                operators,
+                weights,
+                quorum,
+                ..
+            })) => axelar_message_primitives::command::Operators::new(
+                operators,
+                weights.into_iter().map(axelar_message_primitives::command::U256::from).collect(),
+                axelar_message_primitives::command::U256::from(quorum),
```

Comment:
I am mapping the `threshold` with the quorum, but unsure if we are talking about the same thing.

Reply:
yes, they are the same thing 👍